lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -627,4 +630,56 @@ class _MySpaceWidgetState extends State<MySpaceWidget> {
         .onActionClick((data) => mySpaceViewModel.previewDocument(context, document))
         .build();
   }
+
+  String _getIconSort(OrderType orderType) {
```

Comment:
Why not returning the widget directly ?

Reply:
Because the size is different. So I don't think it is necessary to do so in this case.